### PR TITLE
Lilli's edits

### PR DESCRIPTION
**Pull request by lillismith**

### CODE REVIEW

**Comment by lillismith** [comment]
Image 1 in https://github.com/DynamoDS/GenerativePrimer/blob/lillismith-patch-1/1-introduction/1-04_what-goes-into-a-generative-design-process.md is TOOO big and image #2 is too small

File: 1-introduction/1-06_a-further-analogy.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ _above: Citymapper website showing possible routes, considering multiple modes o
 
 To help illustrate this analogy, let's make a table comparing the expected activities when searching for the quickest commute route and what their equivalent would be in a generative design process.
```
Comment (lillismith) [inline 1-introduction/1-06_a-further-analogy.md]: I think this analogy requires a bit more explanation if we're going to use it - in the mapping case we're changing inputs to existing algorithms that are running on the mapping system......in the Refinery case, we're choosing which inputs are going to participate in the study....isn't it pretty different?

File: 1-introduction/1-09_what-is-refinery.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 # What is Refinery?
 
-Project Refinery is Autodesk’s latest generative design application that allows users to explore, evaluate, and optimize their Dynamo designs. Refinery achieves this by allowing users to set multiple \(and sometimes, competing\) design goals, generates a series of feasible solutions, and enables the user to make decisions by automating the creation and evaluation of designs as instructed.
+Project Refinery is Autodesk’s latest application for generative design workflows that allows users to explore, evaluate, and optimize their Dynamo designs. Refinery allows users to set multiple \(and sometimes, competing\) design goals, generates a series of solutions, and enables the user to make decisions by automating the creation and evaluation of designs.
 
-![](../.gitbook/assets/refinery-screenshot.jpg)
+![](../.gitbook/assets/refinery-screenshot.jpg)  - can we show this in color?
```
Comment (radumg) [inline 1-introduction/1-09_what-is-refinery.md]: we should show this in colour

File: 2-getting-started/2-05_refinery-processes.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ To run an optioneering process in Refinery, follow these steps:
 
 ## How to run an optimisation process using Refinery
 
-An optimisation process lets Refinery evolve your design to find the most suitable options based on the constraints and goals provided. Refinery will run multiple generations of options and each time it will take the fittest \(best\) options of the generation and use them to create a new generation. Refinery is using NSGA-II, an elitist multi objective genetic algorithm to optimise results.
+An optimisation process lets Refinery evolve your design to find the most suitable options based on the constraints and goals provided. Refinery will run multiple generations of options and each time it will take the fittest \(best\) options of the generation and use them to create a new generation. Refinery is using NSGA-II <add link to more explanation>, an elitist multi objective genetic algorithm to optimise results.
```
Comment (radumg) [inline 2-getting-started/2-05_refinery-processes.md]: add NSGA-II link

File: 3-optioneering/3-01_what-is-optioneering.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 # What is Optioneering?
 
-![alt text](Images/3-01_what-is-optioneering//3-01_optioneering-example.png "Illustration of crossover (http://mnemstudio.org/genetic-algorithms-recombination.htm)")
+![alt text](Images/3-01_what-is-optioneering//3-01_optioneering-example.png "Illustration of crossover (http://mnemstudio.org/genetic-algorithms-recombination.htm)")  - this image should show filtering and sorting
```
Comment (radumg) [inline 3-optioneering/3-01_what-is-optioneering.md]: this image should show filtering and sorting

**Comment by radumg** [comment]
thanks @lillismith for the comments & edits, we'll make the required updates.

**Comment by radumg** [comment]
> Image 1 in https://github.com/DynamoDS/GenerativePrimer/blob/lillismith-patch-1/1-introduction/1-04_what-goes-into-a-generative-design-process.md is TOOO big and image #2 is too small

We'll fix this. for background info, we had images as HTML `` tags to control sizing but Gitbook advises against, so we changed to `markdown` syntax, but now styling is up to Gitbooks, which isn't great. We'll revert to using HTML.

File: 1-introduction/1-09_what-is-refinery.md (hunk 2 above)
Comment (radumg) [reply 1-introduction/1-09_what-is-refinery.md]: @SHKnudsen

File: 2-getting-started/2-05_refinery-processes.md (hunk 3 above)
Comment (lillismith) [reply 2-getting-started/2-05_refinery-processes.md]: Here's a good one https://www.iitk.ac.in/kangal/Deb_NSGA-II.pdf

**Pull request merged**